lib/JitEngineDevice.hpp

```diff
@@ -512,6 +513,13 @@ void JitEngineDevice<ImplT>::specializeIR(Module &M, StringRef FnName,
         M, *F, RCIndices,
         ArrayRef<RuntimeConstant>{RC,
                                   static_cast<size_t>(NumRuntimeConstants)});
+  
+  if (! getJitVariableMap().empty() ) {
+    for (auto& F : M.getFunctionList()) {
+      if (llvm::demangle(F.getName()).find("lambda") != std::string::npos)
```

Comment:
why don't we match the Symbol (key in the map) to call the transform only if needed?

Reply:
Symbol is of the lambda class, not operator(), so we cannot do a direct match.